CHANGELOG.md

```diff
@@ -9,8 +9,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 
-- MSRV Changed to 1.60
-- Bump `rust-cfg` to 0.5 and `cargo_metadata` to 0.15
+- MSRV Changed to 1.64
```

Comment:
The build matrix was set to 1.72.1? What is the MSRV supposed to be?

Reply:
Missed this one